tests/console/yast2_cmdline.pm

```diff
@@ -40,8 +42,25 @@ sub run_yast_cli_test {
 }
 
 sub run {
+    my $cmd;
+
     select_console 'root-console';
 
+    if (is_sle) {
+        if (is_sle('15+') and get_var('REPO_SLE15_MODULE_BASESYSTEM_SOURCE')) {
```

Comment:
@rwx788 I do not understand why you recommend to extract this part to another file when it is only used within this test module?

Reply:
As mentioned above zypper_info, exactly same thing is required there.

Reply:
ok, then what @rwx788 proposed is fine for me